browser/ui/android/strings/android_brave_strings.grd

```diff
@@ -3642,8 +3642,11 @@ If you don't accept this request, VPN will not reconnect and your internet conne
     <message name="IDS_SPEEDREADER_MESSAGE_BUTTON" desc="The label for the button to enable speedreader on current page.">
       Enable
     </message>
-    <message name="IDS_ENABLE_SPEEDREADER_TITLE" desc="Title for preference that enables or disables Speedreader." translateable="false">
-      Enable Speedreader
+    <message name="IDS_ENABLE_SPEEDREADER_TITLE_1" desc="Translatable part of the title for preference that enables or disables Speedreader.">
+      Enable '''
+    </message>
+    <message name="IDS_ENABLE_SPEEDREADER_TITLE_2" desc="Untranslatable part of the title for preference that enables or disables Speedreader." translateable="false">
```

Comment:
I am not sure I understand why we want to manually construct the title from 2 parts instead of just having one string?

Reply:
Because on Android it is not possible to label part of a string to be untranslatable. To make only `Speedreader` part untranslated, I have to split the whole string into two part and label the second part untranslatable.

Reply:
Oh I see, if we have a sentence with a non-translatable term we don't need to split it up and mark it as non-translatable. In fact, we can't really do this because we don't know the right order of words in all languages. So we should just have one translatable string here.

Reply:
Then I shall just make the whole string translatable?

Reply:
Yes, please. Basically, if a non-translatable term is alone in the string than that string should be marked as non-translatable, but if the term is in a sentence then don't mark the string as non-translatable.

Reply:
I have updated to remove the `translateable="false"` marking for the whole string.